src/web/debug_utils.rs

```diff
@@ -215,9 +217,10 @@ pub fn debug_util_forms(
             br;br;
             form action="/debug_util/make_motion" method="get" {
                 "Make "
-                select name="is_super" {
-                    option value="false" { "simple" }
-                    option value="true" { "super" }
+                select name="power" {
+                    option value="1" { "simple" }
+                    option value="2" { "super" }
+                    option value="0.5" { "sub" }
```

Comment:
Not super important but put these in order

Reply:
Fixed in https://github.com/consortium-chat/plutocradroid/pull/5/commits/2dc71fb4c88858408580c7805606ffa2beccc978